src/dotnet/UI.Blazor/Components/VisualMediaViewerModal/VisualMediaViewerHeader.razor

```diff
@@ -77,4 +91,12 @@
         if (ChatEntry != null)
             _author = await Authors.Get(Session, ChatEntry.ChatId, ChatEntry.AuthorId, CancellationToken.None).ConfigureAwait(false);
     }
+
+    private void SaveImage() {
+        Saver!.Save(DownloadUrl, ContentType);
+        ShowToast();
```

Comment:
It's not needed here. IMedia savers should show toasts internally. We need to figure out why they does not do it now.
I am check the one on Android platform right now.